docs-compose/overwatch_docs/content/FAQ/_index.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
Not sure how this file got in this PR but this should not be going to main and it should not be in this PR. I will remove it for now. When you switch branches, files that are open cannot be removed and thus you can wind up with files lingering between commits / branches. Watch out for this and be sure you close all files before switching branches to protect against this. Before committing / pushing, double check the files changed using `git status` to ensure only the files you want are being committed.